src/StructTree/types.ts

```diff
@@ -0,0 +1,23 @@
+import { PDF_ROLE_TO_HTML_ROLE } from './constants';
+
+export type PdfTagRole = keyof typeof PDF_ROLE_TO_HTML_ROLE;
+
+export type StructTreeNode = {
```

Comment:
Why custom StructTreeNode instead of the one from PDF.js? I was unable to find a node with role, lang and alt, how did you come up with this?

Reply:
The pdfjs web viewer includes the possibility to have role/lang/alt on a struct tree node. I've seen some documents with certain tags where these do show up so it's important to keep them. But the pdfjs struct tree node did not include this stuff. There may be a better way to represent the node we want, maybe with partials. Open to ideas

Reply:
![image](https://github.com/wojtekmaj/react-pdf/assets/24924530/f7f5521a-53ca-45eb-947a-119c33ee32a6)
[SampleJuan.pdf](https://github.com/wojtekmaj/react-pdf/files/11658524/SampleJuan.pdf)

Here's the document in question, with image

Reply:
Oh that is fantastic example, thanks!